pkg/trdm/trdm_test.go

```diff
@@ -0,0 +1,24 @@
+package trdm_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/suite"
+
+	"github.com/transcom/mymove/pkg/testingsuite"
+)
+
+type TRDMSuite struct {
+	*testingsuite.PopTestSuite
+}
+
+func (suite *TRDMSuite) SetupTest() {
+
+}
+func TestModelSuite(t *testing.T) {
+	hs := &TRDMSuite{
+		PopTestSuite: testingsuite.NewPopTestSuite(testingsuite.CurrentPackage(), testingsuite.WithPerTestTransaction()),
+	}
+	suite.Run(t, hs)
+	hs.PopTestSuite.TearDown()
+}
```

Comment:
Would recommend removing empty `SetupTest()` and renaming `TestModelSuite` to `TestTRDMSuite`
CC: @j-Rosenthal